src/core/coreav.cpp

```diff
@@ -113,7 +113,8 @@ bool CoreAV::anyActiveCalls()
 
 bool CoreAV::isCallVideoEnabled(uint32_t friendNum)
 {
-    assert(calls.contains(friendNum));
+    bool contains = calls.contains(friendNum);
+    assert(contains);
     return calls[friendNum].videoEnabled;
```

Comment:
Why create a variable that is never reused?
Additionally, this makes the contains() call run in NDEBUG builds for no reason.